packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -396,14 +396,10 @@ abstract class ProjectBase extends Resource implements IProject {
   }
 }
 
-export interface CommonProjectProps {
-  /**
-   * A description of the project. Use the description to identify the purpose
-   * of the project.
-   *
-   * @default - No description.
-   */
-  readonly description?: string;
+/**
+ * TODO: add properties `queuedTimeoutInMinutes`, `logsConfig`
+ */
+export interface CommonStartBuildOptions {
```

Comment:
I _think_ we can make `CommonStartBuildOptions ` module-private...? (Just remove the `export` here)